supabase_py/lib/Storage/RequestError.py

```diff
@@ -0,0 +1,11 @@
+class RequestError(Exception):
+    """ """
```

Comment:
docstring

Reply:
Yes but i don't know if this is right place to ask but i think the supabase docs should add a list of error codes for the requests and more detail about the api endpoint like all the params etc.....

Reply:
Hi @azvaska, I have docs for the storage-api API here - https://supabase.github.io/storage-api/ I will make it better with the exact error codes in the coming week or two, but the docs should already have information about supported parameters, their format, etc.